### ISSUE TITLE
Move application to specific desktop/space

### ISSUE THREAD

**Issue by henrik242** [jigish/slate#355]
When disconnecting and reconnecting external monitors, I realized that OSX doesn't put windows back where they were.

So, after reconnecting my monitors, is there a way to manually trigger Slate to:
- move Firefox to desktop 3
- move IntelliJ IDEA to desktop 4
- move Terminal to deskop 1
  etc.

I run OSX Mavericks.

Thanks!

**Comment by napcae** [comment]
Hey, did you took a look at the `default` or `layout` directive?

https://github.com/jigish/slate/blob/master/README.md

**Comment by henrik242** [comment]
Yup, but I couldn't figure out how to address the monitors individually. Do I need to know the pixel distance from the primary monitor?  I run a setup with two 24" monitors side by side above my MBP 15" display.

**Comment by Westacular** [comment]
> **Screens**
> 
> Some operations allow you to specify a screen. Here are the list of possible values for screen:
> - Integer representing the screen ID (indexed at 0). Screens are ordered from left to right (by X coordinate of the origin which is the top-left point). If `orderScreensLeftToRight` is set to false, the screen ID is the Mac OS internal ID (indexed at 0). If `orderScreensLeftToRight` is set to false but you still want to reference screens in the default ordered mode, prefix the screen ID with `ordered:`.
> - Screen resolution in the format `WIDTHxHEIGHT` (e.g. `1440x900`)
> - Screen direction relative to the current screen (`left|right|up|above|down|below`)
> - `next` or `previous` (represents the `currentID+1` or `currentID-1` screen)

Assuming the two 24" monitors have the same resolution, you'll need to use the first option to do what you want.
